source/extensions/filters/http/jwt_authn/token_cache.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+#include <chrono>
+#include <memory>
+#include <string>
+
+#include "envoy/common/time.h"
+
+#include "absl/types/optional.h"
+#include "jwt_verify_lib/jwt.h"
+#include "jwt_verify_lib/verify.h"
+#include "simple_lru_cache/simple_lru_cache_inl.h"
+
+using ::google::jwt_verify::Status;
+using ::google::simple_lru_cache::SimpleLRUCache;
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace JwtAuthn {
+
+// The key of the cache is a JWT,
```

Comment:
Cache key is the JWT string,  value is parsed JWT struct

Reply:
Done.